libsemigroups_cppyy/froidure_pin.py

```diff
@@ -22,4 +21,10 @@ def FroidurePin(*args):
     froidure_pin_type = cppyy.gbl.libsemigroups.FroidurePin(types.pop())
     detail.unwrap(froidure_pin_type, froidure_pin_type.factorisation, lambda x:
             list(x))
+    detail.unwrap(froidure_pin_type, froidure_pin_type.left, lambda x:
```

Comment:
You can use `detail.unwrap_int` for each of these three, which auto fills in the lambda function which is the final parameter.